requirements.txt

```diff
@@ -0,0 +1 @@
+cryptography==3.1.1
```

Comment:
I had to add this because of errors I was getting building the docker image (both on my local and on dockerhub)